perf.groovy

```diff
@@ -260,15 +282,18 @@ def static getOSGroup(def os) {
             }
 
             def osGroup = getOSGroup(os)
-            def architecture = 'x64'
-            def configuration = 'Release'
             def runType = isPR ? 'private' : 'rolling'
             def benchViewName = isPR ? 'coreclr private \$BenchviewCommitName' : 'coreclr rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
 
             steps {
                 shell("./tests/scripts/perf-prep.sh")
                 shell("./init-tools.sh")
-                shell("./build.sh ${architecture} ${configuration}")
+                copyArtifacts(fullBuildJobName) {
```

Comment:
Just making sure I understand the logic here.  Does this just pull from the latest completed RHEL build which happens to be the one we triggered?  You can get a handle to the build that you triggered and then pull the artifacts from that.  I recommend doing this.

Here's an example of how this happens in xunit-performance: https://github.com/Microsoft/xunit-performance/blob/master/netci.groovy#L42